tests/test_model_mode.py

```diff
@@ -42,7 +42,7 @@ def test_predict_scores_all_heads(self) -> None:
         # Set into training mode to check if it is correctly set to evaluation mode.
         self.model.train()
 
-        scores = self.model.predict_scores_all_heads(batch)
+        scores = self.model.predict_h(batch)
```

Comment:
@mberr note I changed the name of this function to better reflect it's the counterpart to `model.score_h()`